test/TestProjects/Authoring-Cadl/Generated/Error.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure;
+using Azure.Core;
+
+namespace Azure.Core.Foundations
+{
+    /// <summary> The error object. </summary>
+    public partial class Error
```

Comment:
I expect this would be removed so why it is still shown in the test?

Reply:
Same as above